indexer/src/test/java/org/uniprot/store/indexer/taxonomy/TaxonomyJobIT.java

```diff
@@ -123,7 +123,7 @@ void testTaxonomyIndexingJob() throws Exception {
         assertThat(response, is(notNullValue()));
         assertThat(response.size(), is(2));
 
-        solrQuery = new SolrQuery("content:Sptr_Scientific_5");
+        solrQuery = new SolrQuery("scientific:Sptr_Scientific_5");
```

Comment:
since content => searching over "multiple fields", i think it's good if the actual query is just "Sptr_Scientific_5" now. Because the query-config should have specified `scientific` in it, as one of the fields to search over for default searches.

Same idea should apply to all tests that had a query specifying content. i.e., 
old query = "content:X"
new query = "X"

Reply:
We cannot do it in store